src/components/ChoicesScreen/ChoicesScreen.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {
+  Box,
+  Link,
+  Typography,
+} from '@mui/material';
+import {
+  ChangeHistoryRounded,
+  ClearRounded,
+  InfoOutlined,
+} from '@mui/icons-material';
+import {
+  FlexBoxSpaceBetween,
+} from '../../components/styled/general';
+import {
+  ChoicesBackground,
+  ChoicesPaper,
+  InfoBox,
+} from './ChoicesScreenStyledComponents';
+import { ChoicesCharts } from '../GeneralCharts/ChoicesCharts';
+import { useParams } from 'react-router';
+import { QuizCharts } from '../GeneralCharts/QuizCharts';
+import { GeneralTooltip } from '../GeneralTooltip/GeneralTooltip';
+
+export const ChoicesScreen = ({
+  title = '',
+  type = 'gameChoices',
+  children,
+  gameChoiceValues,
+  userChoices,
+  onKeyDown,
+  onLeft,
+  onRight,
```

Comment:
mmm have the feeling that we might not need to pass these in, can just define these inside this component itself! but can handle it separately later

(also my bad for duplicating those functions across quizzes/game choices to begin with)

Reply:
Actl my bad! We can put those in the component itself since its kind of repeated!